doozerlib/cli/plashet.py

```diff
@@ -501,12 +503,16 @@ def from_tags(config, brew_tag, embargoed_brew_tag, embargoed_nvr, signing_advis
     --brew-tag <tag> <product_version> example: --brew-tag rhaos-4.5-rhel-8-candidate OSE-4.5-RHEL-8 --brew-tag .. ..
     """
     runtime: Runtime = config.runtime
+    runtime.initialize(mode="rpms", clone_source=False, clone_distgits=False, prevent_cloning=True)
+    assembly = runtime.assembly
+    event = runtime.brew_event
     koji_proxy = runtime.build_retrying_koji_client()
     koji_proxy.gssapi_login()
     errata_proxy = xmlrpclib.ServerProxy(config.errata_xmlrpc_url)
 
     if event:
-        event = int(event)
+        if runtime.assembly_basis_event:
+            assert runtime.assembly_basis_event == event  # to be safe
         event_info = koji_proxy.getEvent(event)
     else:
         # If none was specified, lock in an event so that there are no race conditions with
```

Comment:
I wonder if we should simply move this logic into Runtime (i.e. if not part of the assembly and not specified on the command line, just getLastEvent). There no reason for doozer to run without a locked brew event.

Reply:
I am going to move it to Runtime.